install_config/cluster_metrics.adoc

```diff
@@ -99,6 +99,12 @@ for:
 The xref:metrics-deployer[Metrics Deployer] will be discussed in a later step,
 but you must first set up a service account for it:
 
+
+.Grant view permissions to the Hawkular Metrics service:
+----
+oc adm policy add-role-to-user view system:serviceaccount:openshift-infra:hawkular -n openshift-infra
```

Comment:
Can also break this into multiple lines so that it doesn't scroll way long. Like:

```
$ oc adm policy add-role-to-user \
    view system:serviceaccount:openshift-infra:hawkular \
    -n openshift-infra
```

Basically put a space and `\` on the end of a line, and start with like 4 spaces on the next line just so it's more easily readable. When the entire set of lines are copied and pasted, they will all run as a single command with this formatting.